src/routes/OverviewPage/GetStartedCard.tsx

```diff
@@ -3,71 +3,142 @@ import {
+            className="pf-u-mb-md"
+            id="ACS-progress-stepper"
+          >
+            <ProgressStep
+              variant="default"
+              id="ACS-getting-started-step1"
+              titleId="ACS-getting-started-step1-title"
+              aria-label="ACS getting started step 1"
+              icon={1}
+            >
+              <Card isFlat isCompact className="pf-u-h-100 pf-u-m-md">
+                <CardTitle>Get an ACS subscription</CardTitle>
+                <CardBody className="pf-u-pb-0">
+                  Start with a free trial now with a 60-day, self-supported
+                  subscription to Red Hat® Advanced Cluster Security Cloud
+                  Service.
+                </CardBody>
+                <CardFooter>
+                  <Button
+                    component="a"
+                    href="https://www.redhat.com/en/technologies/cloud-computing/openshift/advanced-cluster-security-kubernetes/cloud-service/trial"
+                    target="_blank"
+                    variant="link"
+                    icon={<ExternalLinkAltIcon className="pf-u-ml-xs" />}
+                    iconPosition="end"
+                  >
+                    Start with a free trial
+                  </Button>
+                </CardFooter>
+              </Card>
+            </ProgressStep>
+            <ProgressStep
+              variant="default"
+              id="ACS-getting-started-step2"
+              titleId="ACS-getting-started-step2-title"
+              aria-label="ACS getting started step 2"
+              icon={2}
+            >
+              <Card isFlat isCompact className="pf-u-h-100 pf-u-m-md">
+                <CardTitle>Create an ACS instance</CardTitle>
+                <CardBody className="pf-u-pb-0">
+                  If you&apos;re already subscribed to ACS, continue the process
+                  by setting up an ACS instance to get started with securing
+                  your clusters.
+                </CardBody>
+                <CardFooter>
+                  <Button
+                    component={(props) => (
+                      <AppLink {...props} to={'instances'} />
+                    )}
+                    variant="link"
+                  >
+                    Create ACS instance
+                  </Button>
+                </CardFooter>
+              </Card>
+            </ProgressStep>
+            <ProgressStep
+              variant="default"
+              id="ACS-getting-started-step3"
+              titleId="ACS-getting-started-step3-title"
+              aria-label="ACS getting started step 3"
+              icon={3}
+            >
+              <Card isFlat isCompact className="pf-u-h-100 pf-u-m-md">
+                <CardTitle>Secure a cluster</CardTitle>
+                <CardBody className="pf-u-pb-0">
+                  Once you&apos;ve got your ACS instance up and running, open
```

Comment:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md this lint rule suggests otherwise, which is what prompted me to use the HTML 🤔

Reply:
My mistake:

OPTION + ] is a **left single quote**, which `’` and the lint rule says could be mistaken for a backtick `\``

 The correct key combo is OPTION + SHIFT + ], which results in a **right single quote** `’`.

Might as well party like it's 1999, and leave the HTML entity in there, much as it pains me.